src/tasks/package.js

```diff
@@ -5,7 +5,7 @@ const meta = require('user-meta');
 const gitUsername = require('git-username');
 const { json, install } = require('mrm-core');
 
-const packages = ['@webpack-contrib/schema-utils', 'loader-utils'];
+const packages = ['schema-utils', 'loader-utils'];
```

Comment:
I think we should remove this instead, the dependencies of a paticular package are solely it's own concern. In regrad to `schema-utils` that's mybe ok to install by default as we normally intend to use it everywhere, but please at least definitely remove the `loader-utils`. I'm still in favor of not installing anyting at all, since e.g `webpack-dev-middleware` should use `schema-utils` in the future, but the reality is it currently doesn't and adding just the dependency via a `webpack-defaults` upgrade is basically still _useless_. It's a separate PR to put it in place imho

Reply:
@michael-ciniawsky let's remove `loader-utils` in other PR, this PR only for `schema-utils`

Reply:
Why is another PR needed ? But ok, I can also open another PR to remove this stuff afterwards, if you prefer

Reply:
@michael-ciniawsky yep, removing `loader-utils` maybe need discussion, switch on stable `schema-utils` is another change, here not require discussion